apps/multisig-signer/src/app/components/web3-connector/web3-connector.tsx

```diff
@@ -0,0 +1,94 @@
+import { useEnvironment } from '@vegaprotocol/environment';
+import { useEthereumConfig } from '@vegaprotocol/web3';
+import { Button, Splash, AsyncRenderer } from '@vegaprotocol/ui-toolkit';
+import { Web3ConnectDialog } from '@vegaprotocol/web3';
```

Comment:
chore (blocking): Combine web3 imports

Reply:
After discussion, we will switch this out later: https://github.com/vegaprotocol/frontend-monorepo/issues/1550

Reply:
Oops replied to the wrong comment